SConstruct

```diff
@@ -36,21 +27,9 @@ Export('env')
 SConscript('source/data/SConscript') 
 
 # Additional mode options
-if mode in ['cache', 'release']:
+if mode == 'cache':
```

Comment:
@arosenbe - I've been using `is` in the place of `==` for string comparison, and I've found that `is` often returns `False` when `==` returns `True`. I just tried it out in the template and when `mode=cache` is specified, `mode is 'cache'` returns `False` wereas `mode =='cache'` is `True`. Hence I am reversing this change in `master`.